tox.ini

```diff
@@ -120,3 +124,16 @@ commands =
     coverage: coverage report
     docker-compose: docker-compose {posargs}
     pip-compile: pip-compile {posargs}
+
+[testenv:dev]
```

Comment:
AFAIK this is the only way to set these particular settings for the dev env only.